src/store/conversationsStore.js

```diff
@@ -744,7 +745,18 @@ const actions = {
 			console.error(error)
 			showError(t('spreed', 'Could not set the conversation picture'))
 		}
+	},
 
+	async setConversationEmojiAvatarAction(context, { token, emoji, color }) {
+		try {
+			const response = await setConversationEmojiAvatar(token, emoji, color)
+			const conversation = response.data.ocs.data
+			context.commit('addConversation', conversation)
+			showSuccess(t('spreed', 'Conversation picture set'))
+		} catch (error) {
+			console.error(error)
+			showError(t('spreed', 'Could not set the conversation picture'))
```

Comment:
Should show the error from the response:
https://github.com/nextcloud/spreed/pull/9443/files#diff-d2141d061bc161b347303058b3e5128b4f3dddbd4999da98444d2ef2574d119fR50

Reply:
Actually this should not showError as the UI is already doing it?
And as per recent discussion you should not catch the error here, but the frontend code should deal with it.

Reply:
Picture | Emoji
-- | --
![image](https://user-images.githubusercontent.com/93392545/236424442-fda30985-93fa-41fa-b9b7-404377d08ca6.png) | ![image](https://user-images.githubusercontent.com/93392545/236424548-962da5f9-f1e1-442d-a21a-dec5d14d0b5b.png)